esp/useful_scripts/server_setup/new_site.sh

```diff
@@ -1,20 +1,35 @@
 #!/bin/bash
 
 # ESP site creation script
-# Michael Price, December 2010
+# See /docs/dev/deploying.rst for documentation on how to deploy.
+
+set -e -o pipefail
```

Comment:
In other scripts we've used ``set -euf -o pipefail`` (https://github.com/learning-unlimited/ESP-Website/search?utf8=%E2%9C%93&q=pipefail&type=Code). Should we do that here too for consistency? Briefly glancing through this, it doesn't look like the ``-u`` or ``-f`` flags would break anything in this script.